index.html

```diff
@@ -134,6 +135,8 @@
 			onEachFeature: function(feature, layer) {
 				var popupTxt = "You clicked a great feature!";
 				layer.bindPopup(popupTxt);
+			(feature.properties.highway=residential); return {color: "#24ff5e"};
```

Comment:
@bentneb9 a few things we to add:

1. The "if" statement that will direct the code to evaluate the property of the feature
2. A "catch all" style if the highway is not residential or service.

Take a look at this example:

		style: function(feature) {
			if (feature.properties.building == 'house') {
				return buildingsStyle
			} else if (feature.properties.builing == 'school') {
				return buildingsStyle
			} else {
			return otherStyle
		   }
		}

Reply:
I've updated the bentneb9-Oways branch with these changes